Products/CMFPlone/browser/search.py

```diff
@@ -6,6 +6,7 @@
 from plone.registry.interfaces import IRegistry
 from Products.CMFCore.utils import getToolByName
 from Products.CMFPlone.browser.navtree import getNavigationRoot
+from Products.CMFPlone.interfaces import IPloneSiteRoot
```

Comment:
Interfaces should be imported from `plone.base.interfaces`

Reply:
Fixed in df97eceab7257855935d95e2f2064690857e9e8f.